scripts/cmake-toolchain-script.sh

```diff
@@ -4,8 +4,12 @@ DESTDIR=$1
 PSP_CMAKE_PATH=${DESTDIR}$(psp-config --pspdev-path)/bin
 TOOLCHAIN_SCRIPT_PATH=${DESTDIR}$(psp-config --psp-prefix)/share/cmake
 
-## copy toolchain script and psp-cmake
+## copy toolchain script
 install -d $TOOLCHAIN_SCRIPT_PATH
 install -m644 ../patches/PSP.cmake $TOOLCHAIN_SCRIPT_PATH || { exit 1; }
+## copy CreatePBP.cmake
+install -d $TOOLCHAIN_SCRIPT_PATH
```

Comment:
This line is not needed. The directory only has to be created once.

Reply:
Deleted line